src/themes/oui-cascadia/global_styling/mixins/_button.scss

```diff
@@ -51,6 +51,7 @@
 
   text-decoration: none;
   border: solid 1px transparent;
+  font-weight: 600;
```

Comment:
In the description it says `Set ouiButton font weight to semi-bold`. Where are we changing that?

Reply:
This is that semi-bold change.